src/components/SettingsViewer.tsx

```diff
@@ -0,0 +1,15 @@
+import { useSettings } from "./GlobalSettings";
+
+export default function SettingsViewer() {
+  const { settings } = useSettings(); // Access global settings
+
+  return (
+    <div>
+      <h2>Configuration</h2>
+      <p>Floor Plan: {settings.floorplanImagePath}</p>
```

Comment:
so this shows up as a barebone setting object? 

Ok with me for now, but of course pretty crude

Reply:
SettingsViewer was debugging code. Removed